src/internal/git/utils.go

```diff
@@ -30,14 +30,14 @@ type Credential struct {
 
 var (
 	// For further explanation: https://regex101.com/r/zq64q4/1
```

Comment:
We should make sure we update the regex101 link since we added the `gitPath` capture group.

Reply:
done